Tests/CacheAdvanceTests/CacheAdvanceTests.swift

```diff
@@ -487,8 +595,27 @@ final class CacheAdvanceTests: XCTestCase {
         }
     }
 
+    private func createHeaderHandle(
```

Comment:
Could we consolidate with https://github.com/dfed/CacheAdvance/blob/bfe57b1649a0e3ed2303b8b399cc83206769049b/Tests/CacheAdvanceTests/CacheHeaderHandleTests.swift#L212-L224?

Reply:
In some helper file

Reply:
The signatures are a bit different, since we're doing slightly different things. Thinking there's value in keeping this separate for now.